guake/guake_app.py

```diff
@@ -1350,16 +1350,15 @@ def save_tabs(self, filename="session.json"):
             for index in range(nb.get_n_pages()):
                 try:
                     page = nb.get_nth_page(index)
-                    if page.child:
-                        panes = []
-                        page.save_box_layout(page.child, panes)
-                        tabs.append(
-                            {
-                                "panes": panes,
-                                "label": nb.get_tab_text_index(index),
-                                "custom_label_set": getattr(page, "custom_label_set", False),
-                            }
-                        )
+                    panes = []
```

Comment:
Will this cause regression for 6cc235430f525c999420504e5b49d77bb3669c25?

Reply:
I trimmed the condition because if you check new_page(), we can see that every page has a child so the condition is always true

Reply:
Looks ok for me, do you want to change anything or we can merge it ?

Reply:
I fixed what I found, PR stands for other people to find errors

Reply:
If you've checked the code and tested it and can't find any issues, you can approve so this merges. We don't have to leave every PR in the queue for days